### PR TITLE
Add features command

### PR DESCRIPTION
**Pull request by nadyaA**
Add command that shows account features - which are enabled and which are disabled.

https://github.com/Icenium/fusion-services/pull/128

### CODE REVIEW

File: lib/services/server/server-accounts-service.ts [hunk 1]
```diff
@@ -17,6 +17,11 @@ export class ServerAccountsService extends ServerServiceBase implements IServerA
 		return this.sendRequest<IUsageInfo[]>(HTTP_METHODS.GET, `api/usage?accountId=${accountId}`, null);
 	}
 
+	public getAccountFeatures(accountId: string): Promise<IFeatureInfo[]> {
+		return this.sendRequest<IFeatureInfo[]>(HTTP_METHODS.GET, `api/features?accountId=${accountId}`, null);
+
```
Comment (rosen-vladimirov) [inline lib/services/server/server-accounts-service.ts]: Unnecessary new line

File: lib/definitions/server/server-accounts-service.d.ts [hunk 2]
```diff
@@ -89,3 +92,14 @@ interface IUsageInfo extends IUsageInfoBase {
...
+
+interface IFeatureInfo extends IFeatureInfoBase {
+	/**
+	 * The type of the feature (e.g. CloudBuilds).
+	 */
+	type: string;
+	/**
+	 * If this property is set to true the allowed usage is unlimited.
+	 */
+	enabled: boolean;
```
Comment (rosen-vladimirov) [inline lib/definitions/server/server-accounts-service.d.ts]: this name is really misleading based on the JSDocs above. Is it possible to name it `hasUnlimitedUsage` for example? Ping @TsvetanMilanov

Comment (TsvetanMilanov) [reply lib/definitions/server/server-accounts-service.d.ts]: The name of the property is correct. The JSDocs description should be changed.

**Comment by nadyaA** [comment]
ping @rosen-vladimirov

**Pull request merged**